### DISCUSSION TITLE
Lprint hardware

### DISCUSSION THREAD

**Discussion by milkman0007** [michaelrsweet/lprint#150] [Q&A]
Looking to run lprint on the smallest foot print hardware. Can it be run on MilkV (https://milkv.io/duo) or something similar? I want to make a small print server

**Comment by michaelrsweet** [comment]
I can't really speak to this particular hardware as I haven't tried it. 64MB RAM is probably too little to have a full embedded Linux plus LPrint, but the 256MB versiion *should* be able to handle things. The smallest I've personally tried is the original Pi Zero W - 32-bit ARM + 512MB RAM, but it had plenty of headroom.
